xblock/test/django/test_request.py

```diff
@@ -80,12 +80,12 @@ def test_headers(self):
     def test_content_type(self):
         # Default charset
         self.assertEquals(
-            self._as_django(content_type='application/json')['Content-Type'],
-            'application/json; charset=UTF-8'
+            self._as_django(content_type='text/html')['Content-Type'],
+            'text/html; charset=UTF-8'
```

Comment:
I'm concerned about these test changes.  It seems to be that we are moving from a test that behaves similar to real code, to a test that behaves less like real code.  PR #338 has much more discussion about it.